src/styl/responsive/mobile.styl

```diff
@@ -14,6 +14,14 @@
 
 // ******************************************************
 
+// Splash image
+
+.full-splash-image
+  height: 100vh
+  background-size: cover
```

Comment:
does `cover` no longer center the image? Without adding centering styles it is left aligning?